sympy/simplify/tests/test_simplify.py

```diff
@@ -1080,3 +1078,10 @@ def test_reduce_inverses_nc_pow():
     x, y = symbols("x y", positive=True)
     assert expand((x*y)**Z) == x**Z * y**Z
     assert simplify(x**Z * y**Z) == expand((x*y)**Z)
+
+def test_issue_26086():
```

Comment:
```suggestion

def test_issue_26086():
```

I don't think that you should change the styles of two spacing here, because it is just breaking consistency to all the file.
I also think that people start to prefer two blank lines between functions